examples/get_publicasset.py

```diff
@@ -1,8 +1,12 @@
 """Get a public asset from a instance of Archivist
 """
 
+from warnings import filterwarnings
+
 from archivist.archivistpublic import ArchivistPublic
 
+filterwarnings("ignore", message="Unverified HTTPS request")
```

Comment:
Quick question: what is the purpose of the filter?

Reply:
the urllib library uses the 'warning' package to do logging about doing unverified requests - if this is not set than stdout is littered with warnings which makes things noisy